tensorboard/notebook.py

```diff
@@ -278,9 +278,36 @@ def _display(port=None, height=None, print_message=False, display_handle=None):
 
 
 def _display_colab(port, height, display_handle):
-  # TODO(@wchargin): Implement this after merging this code into
-  # google3, where it's easier to develop and test against Colab.
-  raise NotImplementedError()
+  import IPython.display
```

Comment:
Can you add the commentary from the PR description about how this works to a docstring or something here for posterity?

Reply:
Sure.

Reply:
Can you also add something in the docstring that just generally describes what is necessary?  I.e. "colab has a tunnel on localhost; we set the base href to redirect TB requests to use that tunnel; we have to manually fetch and insert the main HTML because this tunnel cannot intercept `<iframe src>`" etc.

Reply:
Sure.